tests/test_ensemble.py

```diff
@@ -0,0 +1,381 @@
+"""Unit tests for ensemble module."""
+import tempfile
+import unittest
+
+import numpy as np
+
+import relentless
+
+class test_RDF(unittest.TestCase):
+    """Unit tests for relentless.RDF"""
+
+    def test_init(self):
+        """Test creation from data"""
+        #test valid construction
+        r = [1,2,3]
+        g = [2,9,5]
+        rdf = relentless.RDF(r=r, g=g)
+        np.testing.assert_allclose(rdf.table, np.array([[1,2],
+                                                        [2,9],
+                                                        [3,5]]))
```

Comment:
Can you add a quick test that if you call the RDF (as an Interpolator) you get the right value?